### DISCUSSION TITLE
user permissions/group membership to access drawbridge floppy drive on linux?

### DISCUSSION THREAD

**Discussion by Juppstein** [BlitterStudio/amiberry#1180] [Q&A] [answered]
Could please someone point me to what kind of user permissions I need in order to use a drawbridge floppy on linux x86_64 or what groups the user needs to be a member of?  I wasn't able to find that info in the wiki or in the discussions and running amiberry with sudo rights isn't an ideal scenario.

**Comment by giantclambake** [comment]
I think the user would need be a member of the `plugdev`group at a minimum.

**Reply by Juppstein** [reply]
Just tried it, but nope, still get this error window here. So far I tried, plugdev, tty and floppy to no avail.
![Screenshot from 2023-11-17 12-41-43](https://github.com/BlitterStudio/amiberry/assets/8590930/8a7e7acf-7b05-446c-8239-86598629ea5c)

**Reply by midwan** [reply]
It will need access to open, read and write to serial ports in the system. I'm not 100% sure about which groups are required to do that at the moment (it might also differ from distro to distro?).

**Comment by giantclambake** [comment]
In that case, user most likely needs be member of `dialout` group (gives r/w access to ser: )

**Comment by midwan** [comment, marked as answer]
This is now resolved, with the latest versions from both `master` and `preview` branches.
You have the option to select the serial port manually, if Auto-Detect requires permissions and doesn't work right away.
